lang/stack/Portfile

```diff
@@ -6,9 +6,9 @@ PortGroup           gpg_verify 1.0
 PortGroup           haskell_stack 1.0
 PortGroup           legacysupport 1.1
 
-github.setup        commercialhaskell stack 2.7.0.1 rc/v
+github.setup        commercialhaskell stack 2.7.0.3 rc/v
 version             ${github.version}.[lindex [split ${github.tag_prefix} /] 0]
-revision            2
+revision            0
 
```

Comment:
You need to increment the revision.

i don’t know how you are maintaining your local checkouts but git would normally not let you lose other commits like this . Please review your workflow practices to try and avoid this as it is dangerous.

Reply:
Going to close this PR to prevent accidental merging as is. Please reopen once you have review your work practises and fixed the above.

Reply:
No, this is correct. The version is now 2.7.0.3. Before it was 2.7.0.1. The revision for the new 2.7.0.3 is 0, as above.

I simply forgot to do a `selfupdate`.